__tests__/app.test.js

```diff
@@ -31,3 +32,15 @@ describe("GET /api/topics", () => {
     return request(app).get("/api/invalidEndpoint").expect(404);
   });
 });
+
+describe("GET /api/", () => {
+  it("get all valid endpoint", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((res) => {
+        const endpoints = res.body.endpoints;
+        expect(endpoints).toMatchObject(endpoint);
```

Comment:
Nice testing here Rayhan - well done of thinking of requiring in the JSON for comparison  👏🏻